### PR TITLE
Doc update 1.0

### PR DESCRIPTION
**Pull request by jubianchi**

### CODE REVIEW

**Comment by marcofranssen** [comment]
Is this deployment https://jubianchi.gitbook.io/wasmer-docs/v/doc-update-1.0/integrations/examples/instance updated by this PR?

In the deployment I see gaps in the documentation, but here in the markdown I do see some more steps described.

**Review by marcofranssen — CHANGES_REQUESTED** [review]

File: integrations/go/setup.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+mkdir wasmer-project
+cd wasmer-project
+go mod init github.com/$USER/wasmer-project
+```
+
+Now, edit the `go.mod` file to add `wasmer-go` as a dependency:
+
+```bash
+require (
+    require github.com/wasmerio/wasmer-go v1.0.0
```
[the remark this answers is not in the document]
Comment (syrusakbary) [reply integrations/go/setup.md:29]: Shall we create an alpha tag @jubianchi ?

[the remark this answers is not in the document]
Comment (jubianchi) [reply integrations/go/setup.md:29]: @syrusakbary I think we should yes. 

IMHO, we should tag the integrations as soon as we issue the next Wasmer release.

File: integrations/examples/calling-guest-functions.md [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+mkdir wasmer-example-exports-function
+cd wasmer-example-exports-function
+vim Makefile
+```
+
+Let's create a simple `Makefile`:
+
+```c
+CFLAGS = -g -I$(WASMER_C_API)/include
+LDFLAGS = -L$(WASMER_C_API)/lib -Wl,-rpath,$(WASMER_C_API)/lib
```
Comment (syrusakbary) [inline integrations/examples/calling-guest-functions.md:85]: We have the following commands that we can use, please try them out and choose them accordingly:

```
$ wasmer config --cflags
-I/Users/syrus/.wasmer/include/wasmer
```

```
$ wasmer config --includedir
/Users/syrus/.wasmer/include
```

```
$ wasmer config --libdir
/Users/syrus/.wasmer/lib
```

```
$ wasmer config --libs
-L/Users/syrus/.wasmer/lib -lwasmer
```

Comment (jubianchi) [reply integrations/examples/calling-guest-functions.md:85]: way better than what I actually did! will fix that

File: integrations/examples/calling-guest-functions.md [hunk 3]
```diff
@@ -79,9 +79,9 @@ vim Makefile
 Let's create a simple `Makefile`:
 
 ```bash
-CFLAGS = -g -I$(WASMER_C_API)/include
-LDFLAGS = -L$(WASMER_C_API)/lib -Wl,-rpath,$(WASMER_C_API)/lib
-LDLIBS = -lwasmer
+CFLAGS = -g -I$(shell $(WASMER_C_API)/bin/wasmer config --includedir)
```
Comment (syrusakbary) [inline integrations/examples/calling-guest-functions.md:82]: Have you tested this? It should be `WASMER_DIR` not `WASMER_C_API`

**Review by syrusakbary — APPROVED** [review]

File: ecosystem/wasmer/wasmer-features.md [hunk 4]
```diff
@@ -19,7 +19,7 @@ The Wasmer WebAssembly runtime provides various features for users and developer
 | Caching | ✅ | ✅ | ✅ |
 | Emscripten | ✅ | ✅ | ✅ |
 | Metering | ✅ | ✅ | ✅ |
-| Multi-value return | 🔄 | ✅ | ✅ |
+| Multi-value return | ✅ | ✅ | ✅ |
```
Comment (nlewycky) [inline ecosystem/wasmer/wasmer-features.md:22]: I don't think that's right.

File: integrations/js/wasi/untitled.md [hunk 5]
```diff
@@ -0,0 +1,2 @@
+# Untitled
+
```
Comment (nlewycky) [inline integrations/js/wasi/untitled.md:2]: Why is this added?
